hack/loadtest/loadtest.sh

```diff
@@ -16,15 +16,16 @@ NUM_POLICIES=${NUM_POLICIES:-"1000"}
 REQ_COUNT=${REQ_COUNT:-"$NUM_POLICIES"}
 REQ_KIND=${REQ_KIND:-"crs_req01"}
 RPS=${RPS:-"500"}
+SCHEMA_ENFORCEMENT=${SCHEMA_ENFORCEMENT:-"none"}
 STORE=${STORE:-"disk"}
 SERVER=${SERVER:-"localhost:3592"}
 USERNAME=${USERNAME:-"cerbos"}
 PASSWORD=${PASSWORD:-"cerbosAdmin"}
+WORK_DIR=${WORK_DIR:-"work"}
```

Comment:
docker-compose requires a leading `./` on relative paths in bind mounts

```suggestion
WORK_DIR=${WORK_DIR:-"./work"}
```